build/jade.js

```diff
@@ -0,0 +1,56 @@
+var start = Date.now()
+var fs = require('fs')
+var path = require('path')
+var find = require('find')
+var jade = require('jade')
+var async = require('async')
+var mkdirp = require('mkdirp')
+
+var inputDir = path.normalize(path.join(__dirname, '..', 'pages'))
+var outputDir = path.normalize(path.join(__dirname, '..', 'dist'))
+
+find.file(/\index.jade$/, inputDir, (files) => {
```

Comment:
I think you should have a backslash before the dot here instead i.e. `/index\.jade$/`